managed/src/main/java/com/yugabyte/yw/models/Audit.java

```diff
@@ -134,6 +137,17 @@ public static Audit getFromTaskUUID(UUID taskUUID) {
     return find.query().where().eq("task_uuid", taskUUID).findOne();
   }
 
+  public static Audit getOrBadRequest(UUID customerUUID, UUID taskUUID) {
+    Customer.getOrBadRequest(customerUUID);
+    Audit entry =
+        find.query().where().eq("task_uuid", taskUUID).eq("customerUUID", customerUUID).findOne();
```

Comment:
Does this work? `customerUUID` or `customer_uuid`? Not sure ebeans is smart enough to mask the difference.